common-compose.yml

```diff
@@ -0,0 +1,68 @@
+wordpress:
+  build: .
+  privileged: true
+  env_file: _env
+
+nfs:
+    image: autopilot-nfs
+    privileged: true
+    mem_limit: 256m
+    restart: always
+    labels:
+      - triton.cns.services=nfs
+    expose:
+      - 111
+      - 1892
+      - 2049
+    environment:
+      - CONSUL=consul
+
+# MySQL designed for container-native deployment on Joyent's Triton platform.
+
+mysql:
+    image: 0x74696d/triton-mysql:latest
+    mem_limit: 4g
+    env_file: _env
+    environment:
+      - CONTAINERBUDDY=file:///etc/containerbuddy.json
+      - MANTA_PRIVATE_KEY
+
+memcached:
+  image: autopilot-memcached
+  mem_limit: 512m
+  env_file: _env
+  environment:
+    - CONTAINERBUDDY=file:///opt/containerbuddy/config.json
+  ports:
+    - 11211
+
+nginx:
+    image: 0x74696d/triton-nginx
+    mem_limit: 512m
+    ports:
+    - 80
+    env_file: _env
+    environment:
+      - NGINX_CONF
+    command: >
+      /opt/containerbuddy/containerbuddy -config '{"consul":"{{.CONSUL}}:8500","onStart":"/opt/containerbuddy/reload.sh preStart","services":[{"name":"nginx","port": 80,"publicIp": true,"health":"/usr/bin/curl --fail -s http://localhost/health.txt","poll":10,"ttl":25}],"backends":[{"name":"wordpress","poll":5,"onChange":"/opt/containerbuddy/reload.sh"}]}'
```

Comment:
Nginx is running the old version of Containerbuddy today, but will soon expect `preStart` (and emit warnings with `onStart`)